src/algebra/ordered_ring.lean

```diff
@@ -854,13 +843,13 @@ instance to_nonneg_ring : nonneg_ring α :=
       (ne_of_gt (pos_def.1 pb))⟩,
   ..‹linear_nonneg_ring α› }
 
-@[priority 100] -- see Note [lower instance priority]
-instance to_linear_order : linear_order α :=
+local attribute [instance]
+def to_linear_order : linear_order α :=
 { le_total := nonneg_total_iff.1 nonneg_total,
   ..‹linear_nonneg_ring α›, ..(infer_instance : ordered_add_comm_group α) }
 
-@[priority 100] -- see Note [lower instance priority]
-instance to_linear_ordered_ring : linear_ordered_ring α :=
+local attribute [instance]
```

Comment:
```suggestion
/-- Construct `linear_ordered_ring` from `linear_nonneg_ring`.
This is not an instance because we don't use it in `mathlib`. -/
local attribute [instance]
```